calibration/NUTNRB/CGINS-NUTNRB-00260__20130809.csv

```diff
@@ -1,8 +1,8 @@
 serial,name,value,notes
-260,CC_cal_temp,20.04,
+260,CC_cal_temp,20.04, source_file NUTNR-B_ISUS_SN_260_Calibration_Files_2013-08-09.zip > ISUS260B.CAL
```

Comment:
same comment as above for this one...action performed per SW comment:  "Looks good. This replaces CGINS-NUTNRB-00260__20150218.csv in GitHub. That file is this same cal data but with a deployment date in the filename. CGINS-NUTNRB-00260__20150218.csv should be deleted from GitHub."  ACTION by RGT - 20150218 file was deleted but it looks like I renamed the file and updated source_file info.